common/predictive-text/package.json

```diff
@@ -41,10 +41,10 @@
     "mocha-teamcity-reporter": "^2.5.1",
     "sinon": "^7.1.1",
     "ts-node": "^7.0.1",
-    "typescript": "^3.2.1"
+    "typescript": "^3.7.2"
   },
   "dependencies": {
-    "@keymanapp/keyboard-processor": "^14.0.76",
+    "@keymanapp/web-utils": "^14.0.76",
```

Comment:
The number one reason for this PR is right here.

This allows the lm-layer to link in a simple "leaf" package in order to access our KMW string extensions, rather than the whole keyboard-processor package... which then links in _its_ dependencies, etc.  This has generally been the breaking point for long paths on #3098 (deprecated) and #3119 and is currently blocking #3128 from being a clean split.

Refer to https://github.com/keymanapp/keyman/pull/3119#issuecomment-629014680 for an example path - almost all instances where this have arisen (so far) are due to the dependency this one change directly addresses and greatly reduces.

Reply:
This also helps to clean up a number of the `<reference path />` links that were introduced during the refactor, as the most common multi-referenced code is in this package.